plugins/hydra_nevergrad_sweeper/hydra_plugins/hydra_nevergrad_sweeper/_impl.py

```diff
@@ -157,11 +159,32 @@ def sweep(self, arguments: List[str]) -> None:
             )
             self.validate_batch_is_legal(overrides)
             returns = self.launcher.launch(overrides, initial_job_idx=self.job_idx)
-            self.job_idx += len(returns)
             # would have been nice to avoid waiting for all jobs to finish
             # aka batch size Vs steady state (launching a new job whenever one is done)
+            self.job_idx += len(returns)
+            # check job status and prepare losses
+            failures = 0
+            to_tell: List[Tuple[ng.p.Parameter, float]] = []
             for cand, ret in zip(candidates, returns):
-                loss = direction * ret.return_value
+                if ret.status != utils.JobStatus.COMPLETED:
+                    to_tell.append((cand, float("inf")))
```

Comment:
nit, this is a bit cleaner:
```suggestion
                    to_tell.append((cand, math.inf))
```